tests/Unit/ParallelAlgorithms/Interpolation/Test_InterpolatorReceiveAndDumpVolumeData.cpp

```diff
@@ -313,12 +317,34 @@ struct MockMetavariables {
         intrp::callbacks::ObserveTimeSeriesOnSurface<tmpl::list<>,
                                                      InterpolationTargetA>;
   };
+  struct InterpolationTargetB
+      : tt::ConformsTo<intrp::protocols::InterpolationTargetTag> {
```

Comment:
Maybe I'm being pedantic, but what if instead you avoided duplicating code by using a template parameter and if constexpr to decide whether or not to add the interpolating_component ... otherwise, aren't the two targets the same?

Reply:
I intentionally want one target to have the type alias `interpolating_component` and one to not have the type alias. Couldn't do anything with `if constexpr` since they are type aliases, so I just made the second target inherit from the first one.